src/components/document/McfFamilyMeta.tsx

```diff
@@ -0,0 +1,78 @@
+import { ExternalLink } from "@components/ExternalLink";
+import { CountryLink } from "@components/CountryLink";
+import { getIcon } from "@helpers/getMetadataIcon";
+import { mapFamilyMetadata } from "@helpers/mapFamilyMetadata";
+import { TFamilyMetadata, TMCFFamilyMetadata } from "@types";
+
+interface MetadataItemProps {
+  label: string;
+  icon: string;
+  value: string | string[];
+}
+
+type TFamilyMetadataUnion = TFamilyMetadata | TMCFFamilyMetadata;
+
+interface McfFamilyMetaProps {
+  metadata: TFamilyMetadataUnion;
+}
+
+const getCountryLinks = (countryCodes: string[]): JSX.Element[] => {
+  const countries = countryCodes.map((code, index) => (
+    <div className="flex" key={code}>
+      <CountryLink
+        countryCode={code}
+        showFlag={false}
+        showSlugAsChildren={true}
+        className="text-blue-600 underline truncate text-sm text-transform: capitalize"
+      />
+      {index !== countryCodes.length - 1 && <span>,</span>}
```

Comment:
To follow on from the convo above regarding rendering the slug, we can pass the slug into the component here.

Also - we should not include the comma `,` as part of the link

Reply:
Ignore this, comma isn't inside